vision/product_search/product_search_get_similar_products.rb

```diff
@@ -0,0 +1,75 @@
+# Copyright 2018 Google, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START vision_product_search_get_similar_products]
+require "google/cloud/vision"
+
```

Comment:
Look at all of these new lines! There are soooo many! 2, who does do? That's INSANE! Only a crazy person would use 2 spaces! 😉 

🐍

Reply:
One day we'll add this https://github.com/GoogleCloudPlatform/ruby-docs-samples/issues/68